com.etlegacy.ETL.yaml

```
@@ -230,19 +233,23 @@ modules:
           - rm enemy-territory.zip
           - rm et260b.x86_keygen_V03.run
           - rm "README FIRST ! IMPORTANT.txt"
-          # Omni-bot (which is closed source as well)
-          - tar -xf omni-bot.tar.gz
-          - rm omni-bot.tar.gz
+          # All additional components not part of the code base
+          - tar -xf ./etlegacy-v2.76-i386.tar.gz
+          - cp -r ./etlegacy-v2.76-i386/legacy/ .
```

Same, maybe make this a wild card so you don't need to update in the future and only need to change the URL & checksum down below?
```suggestion
          - cp -r ./etlegacy-*-i386/legacy/ .
```

---

I feel that it's better to explicitly type and name these resources, but I don't think we have any hard standard on it.